csrc/device_lower/pass/circular_buffer.cpp

```diff
@@ -505,9 +519,12 @@ class CloneTmaCircularBufferLoopAndInsertSync
         GpuLower::current()->circularBufferInfo().getCircularBufferOptionsFor(
             circular_buffer_loop_->iter_domain());
 
+    Val* linearized_index = GpuLower::current()
+                                ->circularBufferInfo()
+                                .getLinearIndexRelativeForLoopStack(
+                                    for_loop_stack_, insertion_position_);
```

Comment:
nit: this seems to be repeated multiple times. Does it deserve a separate function `linearIndex()`?